### ISSUE TITLE
Is spicetify-nix updated for 2.38 and handling spotify-updates block?

### ISSUE THREAD

**Issue by copeugne** [Gerg-L/spicetify-nix#230]
Hi,

I'm currently using spicetify-nix in my NixOS configuration and noticed that Spicetify CLI has been updated to **version 2.38**. I would like to know if the `spicetify-nix` package is up to date with this version.

Additionally, I noticed that Spicetify now includes the command `spicetify spotify-updates block`. Does spicetify-nix handle this functionality properly?

I'm asking simply because the main maintainer of Spicetify has announced that they will no longer be maintaining it, so no further updates are expected.

Please let me know if further details are needed.
Thanks for maintaining this project!

**Comment by Gerg-L** [comment]
Yeah i saw the announcement, due to how nix works spotify updating itself isn't a possibility so no need to worry about that.
i will however have to pin a spotify version to a compatible one and make sure nixpkgs has the latest spicetify-cli.

i should have some time to work on it this weekend

**Comment by Gerg-L** [comment]
snapcraft has spotify on 1.2.42 on stable, so once 1.2.46 is on stable i'll pin spotify to that.
shouldn't need any changes with spicetify

**Comment by honnip** [comment]
version 2.38.3 is on the unstable channel

```
nix-repl> pkgs.spicetify-cli.version
"2.38.3"
```

**Comment by Gerg-L** [comment]
master is on spotify 1.2.48 which spicetify-cli 2.38.4 supports honnip is workiing on the PR for that
so everything should still be fine

**Comment by Gerg-L** [comment]
spicetify-cli has been updated when needed by their devs. i'm watching this but i doubt it'll be a issue
